src/pages/items/backpacks/index.js

```diff
@@ -129,7 +129,8 @@ function Backpacks(props) {
                             'https://tarkov.dev/images/unknown-item-icon.jpg',
                         name: itemName,
                         price: item.avg24hPrice,
-                        pricePerSlot: Math.floor(item.avg24hPrice / item.slots),
+                        pricePerSlot: item.itemProperties.grid?.totalSize - item.slots !== 0 ? // Prevents divide by zero errors.
+                                      Math.floor(item.avg24hPrice / (item.itemProperties.grid?.totalSize - item.slots)) : 0,
```

Comment:
Do we get a negative price per slot for backpacks that are bigger on the outside than on the inside?

Reply:
Yes, think of it as paying for "negative slots". You would be effectively paying more for fewer inner slots in the case of a VKBO or Sling bag.